package-lock.json

```diff
@@ -5929,8 +5929,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
```

Comment:
I need to check if we need to have this. I see this happening a lot lately with `optional: true` in package-lock.

Reply:
Never mind. Looks like it an Issue with npm.